lib/models/addon.js

```diff
@@ -515,10 +515,15 @@ let addonProto = {
       trees.push(this.jshintAddonTree());
     }
 
-    let mergedTreesForType = mergeTrees(trees, {
-      overwrite: true,
-      annotation: `Addon#treeFor (${this.name} - ${treeType})`,
-    });
+    let mergedTreesForType;
+    if (trees.length) {
```

Comment:
can we be explicit ans use `!== 0` here?

Reply:
actually thinking about it, should we use the fastpath for `length === 1` too?

Reply:
our `mergeTrees` helper already has the `length === 1` passthrough.

Reply:
updated to `!== 0`